book/02-using-atom/sections/02-text.asc

```diff
@@ -3,16 +3,16 @@
 
 While it's pretty easy to move around Atom by clicking with the mouse or using the arrow keys, there are some keybindings that may help you keep your hands on the keyboard and navigate around a little faster.
 
-First of all, Atom ships with many of the basic Emacs keybindings for navigating a document. To go up and down a single character, you can use `ctrl-P` and `ctrl-N`. To go left and right a single character, you can use `ctrl-B` and `ctrl-F`. These are the equivalent of using the arrow keys, though some people prefer not having to move their hands to where the arrow keys are located on their keyboard.
+First of all, Atom ships with many of the basic Emacs keybindings for navigating a document (note that none of these work on Windows). To go up and down a single character, you can use `ctrl-P` and `ctrl-N`. To go left and right a single character, you can use `ctrl-B` and `ctrl-F`. These are the equivalent of using the arrow keys, though some people prefer not having to move their hands to where the arrow keys are located on their keyboard.
```

Comment:
Sure. I was just not sure whether these work on Linux or not, so I opted for mentioning the one platform where they certainly don't know. Do you know? And then I'll reword.